android-lib/src/androidTest/java/com/kinvey/androidTest/model/TestUser.java

```diff
@@ -8,9 +8,8 @@
     @Key("companyName")
     private String companyName;
 
-    public TestUser() {
-
-    }
+    @Key
+    private InternalUserEntity internalUserEntity;
```

Comment:
Do we have a test case where the `User` class is extended?

Reply:
Yes, we have. Previous custom user tests were updated:
https://github.com/Kinvey/java-library/pull/167/files#diff-24be4dde17572f9389b3bbab418535abR378
and
https://github.com/Kinvey/java-library/pull/167/files#diff-24be4dde17572f9389b3bbab418535abR1131